sway/sway-ipc.7.scd

```diff
@@ -337,8 +337,8 @@ node and will have the following properties:
    this, but borders are included.
 |- window_rect
 :  object
-:  The geometry of the contents inside the node. The window decorations are
-   excluded from this calculation, but borders are included.
+:  The geometry of the contents inside the node. The window decorations and
```

Comment:
suggested: "The geometry of the content inside the node. Window decoractions and borders are outside the window_rect."

Reply:
Thank you for taking a look! I agree the description was not great. In [74d42ce](https://github.com/swaywm/sway/pull/7432/commits/74d42ce5d7b964960af47b89701149c8ac1ac13f) I used your suggestion - I think it is an improvement.

I also checked how i3 [describes it](https://i3wm.org/docs/ipc.html):

> window_rect (map)
>
> The coordinates of the **actual client window** inside its container. These coordinates are relative to the container and do not include the window decoration (which is actually rendered on the parent container). So, when using the `default` layout, you will have a 2 pixel border on each side, making the window_rect `{ "x": 2, "y": 0, "width": 632, "height": 366 }` (for example).

And felt that the "relative to the container" part is important, because it is specific only to `window_rect`.

Current version after [7d7d56f](https://github.com/swaywm/sway/pull/7432/commits/7d7d56f941732c15a32db3c2d5e9a11ecd4a3a10):

> The geometry of the content inside the node. These coordinates are relative to the node itself. Window decorations and borders are outside the `_window_rect_`

I'm happy to work on this further if you think we can do better.
Thanks!